docs/MATEKSYS.md

```diff
@@ -28,3 +28,12 @@ Links to the MatekSys website are found below, which include product specificati
 The most convenient way of flashing the mLRS devices is via their USB-C port and using the [mLRS Web Flasher](https://mlrs.xyz/flash) app.
 
 Plug in the USB-C cable while pressing the bind button. This puts the mLRS device into DFU mode (LEDs are not flashing). Then follow the instructions provided by the mLRS Web Flasher app.
+
+## Tx Module | HC-04 Bluetooth Troubleshooting ##
```

Comment:
maybe the Troubleshooting can go, as it also is kind of a normal how to use

Reply:
Title updated